server/graphql/v1/CollectiveInterface.js

```diff
@@ -525,6 +525,7 @@ export const CollectiveInterfaceType = new GraphQLInterfaceType({
       isArchived: { type: GraphQLBoolean },
       isDeletable: { type: GraphQLBoolean },
       host: { type: CollectiveInterfaceType },
+      collective: { type: CollectiveInterfaceType },
```

Comment:
Having `collective.collective` may be confusing. Can we find a more accurate name for this like `hostCollective`, `hostFrontCollective`?

Reply:
I agree, `hostCollective` perhaps.